master/internal/user/postgres_users.go

```diff
@@ -217,51 +191,69 @@ func UpdateUser(
 			return err
 		}
 		if *ug != (model.AgentUserGroup{}) {
-			if err = addAgentUserGroup(context.TODO(), updated.ID, ug); err != nil {
+			if err = addAgentUserGroup(context.TODO(), db.Bun(), updated.ID, ug); err != nil {
 				return err
 			}
 		}
 	}
 
 	if err = tx.Commit(); err != nil {
-		return errors.Wrap(err, "error committing changes to user in UpdateUser")
+		return errors.Wrap(err, "error committing changes to user in Update")
 	}
 
 	return nil
 }
 
-// UserList returns all of the users in the database.
-func UserList(ctx context.Context) (values []model.FullUser, err error) {
-	query, err := os.ReadFile("../../static/srv/list_users.sql")
-	if err != nil {
-		return nil, err
-	}
-
-	err = db.Bun().NewRaw(string(query)).Scan(ctx, &values)
+// List returns all of the users in the database.
+func List(ctx context.Context) (values []model.FullUser, err error) {
+	err = db.Bun().NewSelect().
+		ModelTableExpr("users AS u").
+		Model(&model.User{}).
```

Comment:
do we need this line if we are just using bun as a query builder?

Reply:
but i think we may be better off using the ORM bun features here. check out the first query in `remakeCommandsByType` for an example of what I am talking about.

Reply:
If you're referring to the "Relation" feature, note that in `remakeCommandsByType` we already define this in the relevant struct:
```
	// Relations
	Task       model.Task       `bun:"rel:belongs-to,join:task_id=task_id"`
	Allocation model.Allocation `bun:"rel:belongs-to,join:allocation_id=allocation_id"`
```
I'll be honest ... I don't know how to translate LEFT OUTER JOIN to something here (https://bun.uptrace.dev/guide/relations.htmlhttps://bun.uptrace.dev/guide/relations.html), so this was my hack to preserve (perceived) functionality.

Reply:
Looks like Bun does a LEFT JOIN by default and says to add a NULL check to simulate an INNER JOIN. I think we can just use the Relation feature.